components/base/kinematicbase/kinematics.go

```diff
@@ -20,6 +21,72 @@ type KinematicBase interface {
 	Kinematics() referenceframe.Frame
 }
 
+const (
+	// LinearVelocityMillisPerSec is the linear velocity the base will drive at in mm/s.
+	defaultLinearVelocityMillisPerSec = 100
+
+	// AngularVelocityMillisPerSec is the angular velocity the base will turn with in deg/s.
+	defaultAngularVelocityDegsPerSec = 60
+
+	// distThresholdMM is used when the base is moving to a goal. It is considered successful if it is within this radius.
+	defaultDistThresholdMM = 100 // mm
+
+	// headingThresholdDegrees is used when the base is moving to a goal.
+	// If its heading is within this angle it is considered on the correct path.
+	defaultHeadingThresholdDegrees = 15
+
+	// deviationThreshold is the amount that the base is allowed to deviate from the straight line path it is intended to travel.
+	// If it ever exceeds this amount the movement will fail and an error will be returned.
+	defaultDeviationThreshold = 600.0 // mm
```

Comment:
An upcoming scope will call this `PlanDeviation` so lets rename to `defaultPlanDeviationMM`